rktl_launch/config/global_params.yaml

```diff
@@ -14,11 +14,10 @@ ball:
 
 # car dimensions and physical constants
 cars:
-  width: 0.0762 # (3 in)
-  length: 0.2032 # (8 in)
+  length: 0.12  # wheel center to center
   steering:
-    max_throw: 0.26179938 # 15 degrees
-    rate: 0.52359877 # 30 degrees/s
+    max_throw: 0.1826
```

Comment:
We should probably list units somewhere

Reply:
At the top of the file: `# ALL UNITS IN SI`. I can add them in though.

Reply:
Oh true, didn't see that